spec/services/builders/review_turnaround_spec.rb

```diff
@@ -117,8 +183,19 @@
                pull_request: pr,
                created_at: wednesday_past_week)
       end
+    end
 
-      let!(:review) do
+    shared_examples 'for thursday and two weeks later on wednesday' do
```

Comment:
I think the shared examples name should be something so that when I read
`it behave like x` I understand what it's testing, here you are describing like if it were a `context`